secboot/encrypt_tpm.go

```diff
@@ -29,12 +29,20 @@ var (
 	sbAddRecoveryKeyToLUKS2Container = sb.AddRecoveryKeyToLUKS2Container
 )
 
+const keyslotsAreaKiBSize = 2560 // 2.5MB
```

Comment:
could we use `gadget.Size` here?

Reply:
this is a number of kibytes, not bytes. Also I don't think unless we have a strong reason we should risk circular imports by importing gadget here,  is not a natural relation, at least ATM secboot is a low-level thin wrapper.

Reply:
I guess when possible I prefer to be able to use units than not and I didn't realize there would be a circular import introduced by that

Reply:
I'm not sure it would create a circular import, but is a risk but If we want to use Size more liberally we would have to move it to some *util pkg.